content/general/functions.textile

```diff
@@ -8,17 +8,176 @@ Reactor Functions allow your server-less functions to be invoked following chann
 
 p(tip). Reactor Functions are "rate limited":#transport and designed for low to medium volumes of updates. If you expect a high volume of events and messages (upwards of 20 per second), then you should consider using our "message queues":/general/queues or "firehose":/general/firehose instead as they are designed to scale without limits.
 
-Generally, customers who want to receive events as they happen, use our "message queues":/general/queues or "firehose":/general/firehose, or alternatively our "realtime client libraries":/realtime and subscribe to events and messages published. However, some customers prefer a server-less architecture and want to have their code invoked when a user publishes a message, attaches to a channel, or enters the presence set on a channel. For example, a developer may want to send a welcome message to someone when they first enter a chat channel. A reactor Function can be used to achieve this.
+Generally, customers who want to receive events as they happen, use our "message queues":/general/queues or "firehose":/general/firehose, or alternatively our "realtime client libraries":/realtime and subscribe to events and messages published. However, some customers prefer a server-less architecture and want to have their code invoked when a user publishes a message, attaches to a channel, or enters the presence set on a channel. For example, a developer may want to send a welcome message to someone when they first enter a chat channel. A Reactor Function can be used to achieve this.
+
+Reactor Functions are are designed to trigger server-less functions on common platforms such as Amazon and Google.  If you are looking to trigger the execution of code on your own servers over HTTP when a message is published or event occurs, see our "Reactor WebHooks":/general/webhooks.
+
+h2. Message Format
+
+The format of messages sent can be defined in your "account dashboard":https://support.ably.io/solution/articles/3000048664-how-do-i-access-my-account-dashboard. You can have the message sent as a JSON message, or as a "MessagePack":https://msgpack.org.
+
+* JSON (JavaScript Object Notation): An efficient data-interchange format which is fairly standard and provides simple text based encoding.
+* MsgPack: An efficient binary serialization format that is similar to JSON, but faster and smaller.
```

Comment:
Inconsistent, you refer to MessagePack above and MsgPack here, and this one is not linked to the msgpack.org site.